packages/viewer/src/components/viewer/viewer.tsx

```diff
@@ -196,6 +218,16 @@ export class Viewer {
 
     this.calculateComponentDimensions();
 
+    if (this.streamSessionId == null) {
+      try {
+        this.streamSessionId = getStorageEntry('stream.sessions', entry =>
```

Comment:
Would recommend we prefix this key with `vertexvis` to reduce the chance of naming collisions.